server/requesters/requesterCtrl.js

```diff
@@ -72,17 +72,17 @@ module.exports = {
 
     Transaction.findOne({
       _id: transactionId
-    }, function(err, user) {
+    }, function(err, doc) {
```

Comment:
I would change this to either `transaction` to match the `createTransaction` function, or to `accepted` to match the style of `fulfillTransaction` and `rateHero` functions